packages/story-editor/src/components/form/dateTime/timeZone.js

```diff
@@ -53,15 +53,15 @@ function TimeZone({ date }) {
     return null;
   }
 
-  const { timeZone: timeZoneString } = getOptions();
-  const zoneAbbr = timezone?.length
-    ? format(date, 'T')
-    : `UTC${timeZoneString}`;
-
+  const offsetSymbol = Number( gmtOffset ) >= 0 ? '+' : '';
+  const zoneAbbr =
+    '' !== timezoneAbbr && isNaN( Number( timezoneAbbr ) )
```

Comment:
Why the `isNaN( Number() )` check? 🤔

Reply:
timezoneAbbr can be +04 or GMT. This check ensures that value is a string and not a int.

Reply:
Got it. I see that we do the same thing also in `getTimeZoneString()` (which is the utility providing `timeZone` from `getOptions()` in the `date` package.

I recommend using `Number.isNaN` though instead of `isNaN`.

Reply:
That said, this kinda repeats parts of `getTimeZoneString`. Would be nice if we could reuse that somehow.

For example, we could create a new `getTimeZoneAbbr` util in the date package that takes the abbreviation from the date settings